estate/models/estate_property.py

```diff
@@ -55,3 +67,14 @@ class EstateProperty(models.Model):
         string="Garden Orientation",
         default="north"
     )
+
+    # relational property fields
+    property_type_id = fields.Many2one("estate.property.type", string="Property Type")
+    # this is the buyer id
+    res_partner_id = fields.Many2one("res.partner", string="Buyer", copy=False)
+    # this is the sales id : (Default) the current logged-in user
+    res_users_id = fields.Many2one("res.users", string="Salesperson", default=lambda self: get_current_user())
```

Comment:
Bonne idée mais il existe une approche plus simple. Tu peux retirer la fonction `get_current_user()`
```suggestion
    res_users_id = fields.Many2one("res.users", string="Salesperson", default=lambda self: self.env.user)
```

Reply:
Merci pour l'astuce 🙏